config/autoscaling/production-policy.json

```diff
@@ -1,54 +1,38 @@
 {
-  "instance_min_count": 5,
+  "instance_min_count": 8,
```

Comment:
Since we already have the policy applied, this just changes the minimum quantity to 8, so we can afford to scale down the memory per instance later. 

Fluctuation will be more aggressive going up, which should be safe, and softer on the scale down.
We can then look at the stats after a few day cycles and optimize for cost.